### PR TITLE
Release periodically

### PR DESCRIPTION
**Pull request by VladimirSlavik**
What it says.
Tested piecewise elsewhere.

The one bad thing about cron is that it won't do schedules like "bi-weekly".
Let's talk about the frequency and what days we want this to run etc.

Successful run: https://github.com/VladimirSlavik/anaconda/actions/runs/3291164651/jobs/5424966365

### CODE REVIEW

File: scripts/makebumpver [hunk 1]
```diff
@@ -231,6 +228,10 @@ class MakeBumpVer:
 
     def _queryBug(self, bugid):
         if not self.bz:
+            sys.stdout.write("Importing bugzilla module...")
```
Comment (jkonecny12) [inline scripts/makebumpver:231]: I don't think we want it logged.

**Review by jkonecny12 — APPROVED** [review]
Looks good to me except the nitpick.

File: scripts/makebumpver (hunk 1 above)
Comment (VladimirSlavik) [reply scripts/makebumpver:231]: Fair enough. Maybe we could drop more of the logging too...

**Comment by VladimirSlavik** [comment]
Rebased - removed the debug message, added more comments.

**Pull request merged**